docs/genai/deployment/gpu_utilization.md

```diff
@@ -0,0 +1,42 @@
+(gpu-utilization)=
+# GPU utilization
```

Comment:
I think this page is important and gives a very good list of options, but is somewhat theoretic - I would expect some examples of how to do things, or at least links to other places where these demos exist. A lot of these techniques are utilized in the GenAI tutorials that we have - we can point there for references.

Reply:
Agree, we can do that next iteration